presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestApproximatePercentileAggregation.java

```diff
@@ -34,14 +47,37 @@
 import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
 import static com.facebook.presto.spi.type.RealType.REAL;
 import static com.facebook.presto.sql.analyzer.TypeSignatureProvider.fromTypes;
+import static com.facebook.presto.tdigest.StatisticalDigestImplementation.QDIGEST;
+import static com.facebook.presto.tdigest.StatisticalDigestImplementation.TDIGEST;
+import static java.lang.Integer.min;
+import static org.testng.Assert.assertTrue;
 
 public class TestApproximatePercentileAggregation
```

Comment:
I would prefer to keep the tests separate and only resort to the non-deterministic check with delta when we must (which is for T-Digest).

Reply:
We actually don't even need it. I used different parameters when creating the distribution that we're testing, and values don't change when we run it with offsets.